### PR TITLE
PHR-10238 Add go implementation of test-control server.

### PR DESCRIPTION
**Pull request by MFAshby**
Extracted http server code from https://github.com/patientsknowbest/docker/tree/master/postgres-test-base/test-control

Defines a module which can be used in applications written in go so they can participate in test-control.

### CODE REVIEW

File: test-control-go/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+)
+
+type myTestControl struct{}
+
+/// Implement TestControl interface for myTestControl
+/// func SetNamespace...
+///
+
+func main() {
+	go log.Fatal(testcontrol.RunTestControl(":9876", "http://test-control:9876", "foo", "http://foo:9876", myTestControl{}))
```
[the remark this answers is not in the document]
Comment (MFAshby) [reply test-control-go/README.md]: fixed

**Review by pkbdevdeploy — COMMENTED** [review]
Version `58-head-579aadd-229484-SNAPSHOT` was generated from commit `579aadd9a2d995dd5fa0ac2dde4ea3f0aed51ca2`

**Review by utamas — APPROVED** [review]

**Review by homps — APPROVED** [review]

**Review by pkbdevdeploy — COMMENTED** [review]
Version `58-129dd85-229811-SNAPSHOT` was generated from commit `129dd852cb358a9c23427d558ee5c034ed27ee6b`

**Review by pkbdevdeploy — COMMENTED** [review]
Version `58-129dd85-229817` was generated from commit `129dd852cb358a9c23427d558ee5c034ed27ee6b`

**Pull request merged**